libs/features/mas/test/merch-quantity-select.test.html.js

```diff
@@ -189,7 +189,7 @@ runTests(async () => {
                 });
                 event.composedPath = () => [quantitySelect];
                 inputField.dispatchEvent(event);
-                await delay();
+                await delay(600);
```

Comment:
In regards to all of these, is 600 the minimum ms you can delay? If i am reading this correctly, this will add 2.4s to our testing suite.